### ISSUE TITLE
Network not learning during training 

### ISSUE THREAD

**Issue by ViktoriaO1** [ildoonet/tf-pose-estimation#436]
Hi @ildoonet ,
thanks a lot for your great work !

I have a question related to [luckhhn's question](https://github.com/ildoonet/tf-pose-estimation/issues/390#issuecomment-472342112). I have tried to replicate the results you have shown in the demo part of the repo with both the cmu and vgg pretrained network weights. I also downloaded the Coco17 training and validation datasets and annotations, so technically I should have followed all the recommended steps to start the training. 
The training is running and there are no errors, but the losses look bad and also the sample output every 2000 iterations show no learning.
I have checked the maps for the PAFs and the heatmaps, but essentially the network only learns what I suspect is the mean of the ground truth PAF vector fields and heatmaps.

My parameters are as follows:

- GPU: Nvidia GeForce RTX 2080Ti
- batchsize: 16 (also can't go over this batchsize due to memory constraints)
- learning rate: 0.001 (the default)
- everything else is also left at default values...

I've tried to get some reasonable results now many times, but unfortunately always without luck. I would be very grateful for any advice! 
Thanks a lot in advance !

I have attached some loss and testing results from when I started the training with the VGG setting and weights. It looks pretty much the same for the CMU case.. (Note that the training also takes very long - the x axis scale is in hours and at 18 hours it had finished approx 45500 iterations or 6 epochs)

![Screenshot from 2019-04-12 09-56-29](https://user-images.githubusercontent.com/28260796/56022194-74604380-5d0b-11e9-84e1-b595420790d5.png)
![Screenshot from 2019-04-12 09-57-11](https://user-images.githubusercontent.com/28260796/56022202-7a562480-5d0b-11e9-8381-883820daff50.png)
![Screenshot from 2019-04-12 09-57-35](https://user-images.githubusercontent.com/28260796/56022215-7d511500-5d0b-11e9-859a-bc881d3ea0aa.png)

**Comment by ninesky110** [comment]
Hi  ,I also have the same problem,have you solved this problem?

**Comment by ViktoriaO1** [comment]
No unfortunately not. I tried a couple of more times and then abandoned it. Would be happy to try again, though, should anyone be able to help...

**Comment by aspenlin** [comment]
Hi @ViktoriaO1 , may I ask which program you used to plot the loss plot? I didn't find any explicit description anywhere. Thanks for your help.

**Comment by aspenlin** [comment]
Sorry for the interruption. I understand that it was plotted by Tensorboard now. Thanks.

**Comment by pannaf** [comment]
I'm also experiencing the same issue. Has anyone been able to reproduce the reported training results?

**Comment by xiaopingxiao** [comment]
Hello, an error has occurred when I was in training module 'tensorflow. Python. Training. Training' has no attribute 'cosine_decay'.I haven't been able to get the training result, and many parameters are similar to yours. May I ask if there is any problem that I missed? Could you please communicate with me
